src/Draco.Compiler.Tests/Syntax/ParserTests.cs

```diff
@@ -186,6 +186,18 @@ public void TestEmptyFuncWithoutClosingCurly()
         }
     }
 
+    [Fact]
+    public void TestFloatingPointLiteralMissingValueAfterExponent()
+    {
+        this.ParseExpression("""
+            0.5e
+            """);
+        this.N<Expr.Literal>();
+        {
+            this.MissingT(TokenType.LiteralFloat);
```

Comment:
Why is it a missing token?

Reply:
I suspect you mean to check that it has diagnostics here, but the naming threw me off

Reply:
I might suggest removing the test as a whole, as it doesn't test anything in the parser